docs/getting_started/install_with_ddev.md

```diff
@@ -0,0 +1,405 @@
+  --ip='*' \
+  --host-name='my-ddev-project.ddev.site' \
+  --host-alias='*.ddev.site' \
+  --basedir='/var/www/html' \
+  --sf-env=dev \
+  > .ddev/apache/apache-site.conf
+sed -i 's/php5-fpm.sock/php-fpm.sock/' .ddev/apache/apache-site.conf
+rm vhost.sh
+```
+
+Then, restart the project:
+
+```bash
+ddev restart
+```
+
+### Run an already existing project
+
+To run an existing project, you need to:
+
+1. Configure the DDEV project.
+1. Start the DDEV project.
+1. Add Composer authentication.
+1. Install dependencies packages with Composer.
+1. Populate the contents, which could mean:
+    - getting a clean database with ddev `php bin/console ibexa:install ibexa-<edition>` and adding some data with [Ibexa data migration](../content_management/data_migration/importing_data.md), or
+    - injecting a dump with [`ddev import-db`](https://ddev.readthedocs.io/en/latest/users/usage/commands/#import-db) and copying related binary files into `public/var`.
+
+The following examples run an already [version-controlled project](install_ibexa_dxp.md#add-project-to-version-control) and have the right content structure (but no content):
+
+```bash
+# Clone the version-controlled project and enter its local directory
+git clone <repository> my-ddev-project && cd my-ddev-project
+# Exclude the whole `.ddev/` directory from version control
+.ddev/ >> .gitignore
+# Configure the DDEV project then start it
+ddev config --project-type=php --php-version 8.1 \
+  --docroot=public \
+  --web-environment-add DATABASE_URL=mysql://db:db@db:3306/db \
+  --http-port=8080 --https-port=8443
+ddev start
+# Configure Composer authentication
+ddev composer config --global http-basic.updates.ibexa.co <installation-key> <token-password>
+# Install the dependencies packages
+ddev composer install
+# Populate the database with a clean install
+ddev php bin/console ibexa:install ibexa-<edition>
+# Add some content types using a migration file (previously created on another installation) and update the GraphQL schema
+ddev php bin/console ibexa:migrations:migrate --file=project_content_types.yaml
+ddev php bin/console ibexa:graphql:generate-schema
+# Open the project in the default brother which should display the default SiteAccess frontpage
+ddev launch
+```
+
+Notice that the example adds the whole `.ddev/` directory to `.gitignore`, but you can also version parts of it.
+Some DDEV configs can be shared among developers. For example, a common `.ddev/config.yaml` can be committed for everyone and [locally extended or overridden](https://ddev.readthedocs.io/en/latest/users/extend/customization-extendibility/#extending-configyaml-with-custom-configyaml-files).
+
+Compared to running a clean install like described in [Installation steps](#installation-steps), you can proceed as follows:
+
+- In [1. Create a DDEV project directory](#1-create-a-ddev-project-directory), you can use an existing directory that contains an Ibexa DXP project instead of creating an empty directory.
```

Comment:
```suggestion
- In [1. Create a DDEV project directory](#1-create-a-ddev-project-directory), you can use an existing directory that contains an [[= product_name =]] project instead of creating an empty directory.
```